jboss-tck-runner/src/test/tck20/tck-tests.xml

```diff
@@ -92,6 +92,84 @@
+                </methods>
+            </class>
+
+            <!-- WELD-2268 -->
+            <class name="org.jboss.cdi.tck.tests.deployment.trimmed.TrimmedBeanArchiveTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2265 -->
+            <class name="org.jboss.cdi.tck.tests.extensions.afterBeanDiscovery.AfterBeanDiscoveryTest">
+                <methods>
+                    <exclude name="testProcessBeanIsFired"/>
+                </methods>
+            </class>
+            <class name="org.jboss.cdi.tck.tests.extensions.configurators.bean.BeanConfiguratorTest">
+                <methods>
+                    <exclude name="processSynthethicBeanEventFired"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2257 -->
+            <class name="org.jboss.cdi.tck.tests.extensions.interceptionFactory.InterceptionFactoryTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+            <class name="org.jboss.cdi.tck.tests.extensions.interceptionFactory.broken.InvalidInterceptionFactoryInjectionPointTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+            <class name="org.jboss.cdi.tck.tests.extensions.interceptionFactory.broken.UnproxyableInterceptedInstanceTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2271 -->
+            <class name="org.jboss.cdi.tck.tests.extensions.lifecycle.broken.observerMethod.ObserverMethodWithoutNotifyMethodTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2263 -->
+            <class name="org.jboss.cdi.tck.tests.extensions.lifecycle.processBeanAttributes.ignoreFinalMethods.IgnoreFinalMethodsTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2264 -->
+            <class name="org.jboss.cdi.tck.tests.extensions.configurators.annotatedTypeConfigurator.beforeBeanDiscovery.AnnotatedTypeConfiguratorInBBDTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2267 -->
+            <class name="org.jboss.cdi.tck.tests.context.activation.ActivateRequestContextinEETest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
+            <!-- WELD-2269 -->
+            <class name="org.jboss.cdi.tck.tests.context.request.event.RequestScopeEventTest">
+                <methods>
+                    <exclude name=".*"/>
+                </methods>
+            </class>
+
```

Comment:
I think it's missing SE tests. I can see following failures locally:
- InvalidContextualReferenceTest
- ContextSETest
- ActivateRequestContextByInterceptorTest
- ActivateRequestContextProgrammaticallyTest.